rosdep/python.yaml

```diff
@@ -2564,7 +2564,9 @@ python-mako:
   debian: [python-mako]
   fedora: [python-mako]
   gentoo: [dev-python/mako]
-  ubuntu: [python-mako]
+  ubuntu:
+    '*': [python3-mako]
+    bionic: [python-mako]
```

Comment:
Rather than mixing Python 2 and Python 3 into the same rule, we prefer to create a separate rules for each. Instead of modifying this rule, please create a new rule for `python3-mako` ([see guideline](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#python-3)).

Reply:
OK, I am leaving the original key untouched and adding a `python3-mako` key. I copied over the fedora and gentoo entries from the `python-mako` key but am not sure if the naming is correct for the following packages that I found:
- https://packages.fedoraproject.org/pkgs/python-mako/python3-mako/
- https://packages.gentoo.org/packages/dev-python/mako